chains/solana/contracts/programs/ccip-router/src/instructions/v1/onramp.rs

```diff
@@ -119,6 +119,17 @@ pub fn ccip_send<'info>(
         &per_chain_per_token_config_accounts,
     )?;
 
+    let link_fee = fee.convert(
```

Comment:
Looking at the EVM code, there's also a `feeValueJuels` in the `EVM2AnyRampMessage` struct, which is what ultimately is emitted in the onramp event. I see that in Solana we don't have an equivalent property, but it feels like we should probably add it to the `Solana2AnyRampMessage`, wdyt?

Reply:
Yep, there's a ticket for that which I'll tackle after this one: https://smartcontract-it.atlassian.net/browse/NONEVM-1147